types/app.d.ts

```diff
@@ -43,10 +43,7 @@ declare global {
    * Its hack way to export redux infering types from @/app
```

Comment:
Fixed a typographical error in the comment. It is better this way:
```
// ⚠️ FSD
// This is a hacky way to export Redux types inferred from @/app
// and use them in @/shared/model/hooks.ts
```

Reply:
@GreenRoot Thanks for suggestion, fixed in https://github.com/noveogroup-amorgunov/nukeapp/pull/25/commits/86db6441dcf4a943b3520358e6678a35bdbd57a6